yellowbrick/features/pcoords.py

```diff
@@ -53,9 +53,9 @@ def parallel_coordinates(X, y, ax=None, features=None, classes=None,
         The axis to plot the figure on. If None is passed in the current axes
         will be used (or generated if required).
 
-    features : list, default: None
-        a list of feature names to use
-        If a DataFrame is passed to fit and features is None, feature
+    labels : list, default: None
+        a list of label names to use
```

Comment:
A list of feature labels to use
If a DataFrame is passed to fit and labels is None, feature 
labels are selected as columns of the DataFrame.

Reply:
This is actually the most sensical use of the `labels` and was the motivating factor in #313